front/components/ContentNodeTree.tsx

```diff
@@ -121,6 +122,7 @@ function ContentNodeTreeChildren({
   parentId,
   parentIds,
   parentIsSelected,
+  additionalActions = undefined,
```

Comment:
this component is used in a lot of places, so I added a new `additionalActions` to pass what I want